website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Configure exposures automatically"
+sidebar_label: "Configure exposures automatically"
+description: "Configure and visualize exposures automatically to auto-generate them from Tableau dashboards, helping you understand how models are used in downstream tools for a richer downstream lineage"
+image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
+---
+
+# Configure exposures automatically <Lifecycle status="enterprise" />
+
+<IntroText>
+Configure and automatically populate downstream exposures for supported BI tool integrations. Visualize and orchestrate them through <a href="https://docs.getdbt.com/docs/collaborate/explore-projects">dbt Explorer</a> and the [dbt Cloud job scheduler](/docs/deploy/job-scheduler) for a richer experience.
```

Comment:
We say "supported BI tool integrations" here and I assume the list will grow, but the description and parts below only mention Tableau. Maybe we should specify Tableau here until other tools are supported or switch to this generic language in the description, too.

Reply:
good shout! added a tableau